guide/06-imagery-and-raster-analysis/creating_imagery_layers.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<p>The <a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.analytics.html#copy-raster" target="_blank">copy_raster</a> function also enables you <strong>to</strong> create an imagery layer from just one image. This option supports all supported raster and image formats, multidimensional raster data, and configuration with various source types and compression settings. Use this option for simple imagery layers where advanced metadata is not needed.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>